docs/additional-functionality/shuffle-docker-examples/Dockerfile.rocky_no_rdma

```diff
@@ -33,11 +33,11 @@ ARG UCX_VER
 ARG UCX_CUDA_VER
 ARG UCX_ARCH
 
-RUN yum update -y && yum install -y wget bzip2 numactl-libs libgomp
+# note that libibmad is a temporary workaround for a missed dependency in ucx-1.16 rpm
```

Comment:
Is there an issue to fix this? Or to move to a non-rc version of UCX?

Reply:
It will be included in the next RC for UCX. The UCX team is working on it here: https://github.com/openucx/ucx/pull/9628.

Reply:
small precision, `libibmad`/`libibumad` dep as just been removed by openucx/ucx#9625 and openucx/ucx#9627. other PRs are longer term effort.